PDE/advection1reverse.m

```diff
@@ -0,0 +1,103 @@
+function pde = advection1reverse
+%1D Test case of the 1D advection equation with
+% the direction of motion reversed from advection1.m.
+% Code designed to test inhomogeneous Dirichlet BC in conjunction with
+% advection1.m
```

Comment:
Perhaps change the descriptive comment to be ...

```
% 1D advection problem with inhomogeneous Dirichlet BC (flow direction reversed from advection1.m)
```

Reply:
The descriptive comment was added.